<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogEsiti</class>
 <widget class="QDialog" name="DialogEsiti">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget#DialogEsiti{
background-color:qlineargradient(spread:pad, x1:0.058, y1:0.136364, x2:1, y2:1, stop:0 rgba(0, 255, 127, 255), stop:1 rgba(255, 255, 255, 255));}</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>160</y>
     <width>1041</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 36pt &quot;MS Shell Dlg 2&quot;;color:rgb(255, 255, 255);
font-weight:bold;
</string>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;ARCHIVIO DEGLI ESITI&lt;/p&gt;&lt;p align=&quot;center&quot;&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>-100</x>
     <y>370</y>
     <width>571</width>
     <height>441</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>img/covidtesting.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QTableWidget" name="EsitiTable">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>250</y>
     <width>761</width>
     <height>381</height>
    </rect>
   </property>
   <column>
    <property name="text">
     <string>Codice</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>CF</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Data</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Stato</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Esito</string>
    </property>
   </column>
  </widget>
  <widget class="QPushButton" name="homeBtn">
   <property name="geometry">
    <rect>
     <x>1070</x>
     <y>670</y>
     <width>71</width>
     <height>51</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>img/home.png</normaloff>img/home.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>30</width>
     <height>30</height>
    </size>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1271</width>
     <height>231</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>img/farmaciaa.jpg</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="showdate_2">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>630</y>
     <width>171</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>MS Shell Dlg 2</family>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <italic>false</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 10pt &quot;MS Shell Dlg 2&quot;;
font-weight:bold;</string>
   </property>
   <property name="text">
    <string>Mostra grafici </string>
   </property>
  </widget>
  <widget class="QComboBox" name="graficiesitiCombo">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>640</y>
     <width>171</width>
     <height>41</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string/>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Esiti</string>
    </property>
   </item>
  </widget>
  <widget class="QLineEdit" name="ricercaappuntamento">
   <property name="geometry">
    <rect>
     <x>730</x>
     <y>640</y>
     <width>211</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="showdate_3">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>630</y>
     <width>71</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>MS Shell Dlg 2</family>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <italic>false</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 10pt &quot;MS Shell Dlg 2&quot;;
font-weight:bold;</string>
   </property>
   <property name="text">
    <string>Ricerca</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Ricercaappuntamentobtn">
   <property name="geometry">
    <rect>
     <x>950</x>
     <y>640</y>
     <width>61</width>
     <height>41</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>img/lens.png</normaloff>img/lens.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>30</width>
     <height>30</height>
    </size>
   </property>
  </widget>
  <zorder>label_2</zorder>
  <zorder>EsitiTable</zorder>
  <zorder>homeBtn</zorder>
  <zorder>label_3</zorder>
  <zorder>label</zorder>
  <zorder>showdate_2</zorder>
  <zorder>graficiesitiCombo</zorder>
  <zorder>ricercaappuntamento</zorder>
  <zorder>showdate_3</zorder>
  <zorder>Ricercaappuntamentobtn</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
